src/app/test/test.component.html

```diff
@@ -5,79 +5,93 @@
-      <app-button data-cy-test="refresh" [icon]="'fa fa-refresh'" [title]="'ReloadTestReports'" (click)="loadData(null)"></app-button>
+      <app-button data-cy-test="refresh" [icon]="'fa fa-refresh'" [title]="'ReloadTestReports'"
+                  (click)="loadData(null)"></app-button>
       <app-button [text]="'Reset'" [title]="'Reset'" (click)="resetRunner()"></app-button>
       <app-button [icon]="'fa fa-cog'" [title]="'Options'" (click)="openSettingsModal()"></app-button>
-      <app-button data-cy-test="selectAll" [title]="'Select All'" [text]="'Select All'" (click)="checkAll()"></app-button>
-      <app-button data-cy-test="deselectAll" [title]="'Deselect All'" [text]="'Deselect All'" (click)="uncheckAll()"></app-button>
-      <app-button data-cy-test="deleteSelected" [icon]="'fa fa-trash'" [title]="'Delete Selected'" (click)="openDeleteModal()"></app-button>
+      <app-button data-cy-test="selectAll" [title]="'Select All'" [text]="'Select All'"
+                  (click)="checkAll()"></app-button>
+      <app-button data-cy-test="deselectAll" [title]="'Deselect All'" [text]="'Deselect All'"
+                  (click)="uncheckAll()"></app-button>
+      <app-button data-cy-test="deleteSelected" [icon]="'fa fa-trash'" [title]="'Delete Selected'"
+                  (click)="openDeleteModal()"></app-button>
       <app-button [text]="'Clone'" [title]="'Clone report'" (click)="openCloneModal()"></app-button>
-      <app-button data-cy-test="copySelected" [icon]="'fa fa-copy'" [title]="'Copy Selected'" (click)="copySelected()"></app-button>
-      <app-button data-cy-test="downloadBinary" [icon]="'fa fa-download'" [title]="'Download Binary'" (click)="downloadSelected()"></app-button>
+      <app-button data-cy-test="copySelected" [icon]="'fa fa-copy'" [title]="'Copy Selected'"
+                  (click)="copySelected()"></app-button>
+      <app-button data-cy-test="downloadBinary" [icon]="'fa fa-download'" [title]="'Download Binary'"
+                  (click)="downloadSelected()"></app-button>
       <button data-cy-test="upload" type="button" class="btn btn-info my-2 mx-1 px-3"
               title="Upload" (click)="uploadFileTest.click()">
         <i class="fa fa-upload"></i>
-        <input data-cy-test="uploadFile" #uploadFileTest (change)="uploadReport($event)" type="file" id="uploadFileTest">
+        <input data-cy-test="uploadFile" #uploadFileTest (change)="uploadReport($event)" type="file"
+               id="uploadFileTest">
       </button>
     </div>
 
 
     <div class="form-group mt-2">
       <label for="moveToInput">Move/Copy to:</label>
       <form class="input-group">
-        <input type="text" class="form-control mr-1 my-2 py-0" id="moveToInput" name="filter" value="{{currentFilter}}"/>
+        <input type="text" class="form-control mr-1 my-2 py-0" id="moveToInput" name="filter"
+               value="{{currentFilter}}" />
```

Comment:
(Jaja niet jouw code)
Prefer binding over interpolation

Reply:
This is the what the querySelector is used for that you've commented about earlier, so it will be fixed the same way.